source/extensions/filters/http/fault/fault_filter.cc

```diff
@@ -441,6 +442,7 @@ Http::FilterTrailersStatus FaultFilter::encodeTrailers(Http::ResponseTrailerMap&
 }
 
 StreamRateLimiter::StreamRateLimiter(uint64_t max_kbps, uint64_t max_buffered_data,
+                                     std::function<void()> overflow_cb,
```

Comment:
overflow cb seems to be the last argument most of the time.  Why is it the first in this case?

Reply:
Thanks for pointing it out... Actually it seems that it is reversed compared to others. Usually it is low_cb, high_cb, overflow_cb.
Not sure why, and I'll try to find out.